pkg/placement/placement.go

```diff
@@ -234,16 +258,18 @@ func (p *Service) ReportDaprStatus(stream placementv1pb.Placement_ReportDaprStat
 				continue
 			}
 
+			now := p.clock.Now()
+
 			for _, entity := range req.Entities {
-				monitoring.RecordActorHeartbeat(req.Id, entity, req.Name, req.Pod, p.clock.Now())
+				monitoring.RecordActorHeartbeat(req.Id, entity, req.Name, req.Pod, now)
```

Comment:
is this a blocking call which takes some time, and if so should `now` be calculated again after the loop?